### PR TITLE
Update 16_woman_cancer.do

### PR DESCRIPTION
**Pull request by sneelsen**

### CODE REVIEW

File: 16_woman_cancer.do [hunk 1]
```diff
@@ -46,7 +46,7 @@ if _rc==0 {
 gen w_mammogram_ref = "" 
 gen w_papsmear_ref = ""
 //if not in adeptfile, please generate value, otherwise keep it missing. 
-//if there are several recall periods, use the shorter one. 
+//if the preferred recall is not available (3 years for pap, 2 years for mam) use shortest other available recall
```
Comment (wengxyu1030) [inline 16_woman_cancer.do:49]: Great, that makes more sense! Thanks! I am merging this change (branch) to the master file.

**Comment by wengxyu1030** [comment]
Great!  I just made comments on the specific code line, this is the comment reacting to the overall file change.

**Comment by sneelsen** [comment]
Ok thanks! Are you always notified when I make a change?

Sven Neelsen, PhD
Economist
HNP Global Engagement
T  +1 (202) 473 0262
M  +1 (202) 597 8006
A 1776 G Street NW, G7-080, Washington, DC 20433 USA
[cid:image002.jpg@01D61E38.70E9F130]

**Comment by wengxyu1030** [comment]
Every time you make changes, the changes firstly saved on your branch (the snapshot of the master version with your modifications saved as an independent version), which is step 1. This step I will not be notified.

Then, if you are confident with your changes and want me to merge it to the master version, which is managed by the author (me).  You should pull the request, as I implicated in the email as step 2, this is the time when I will be notified.

**Comment by sneelsen** [comment]
Thanks!

Sven Neelsen, PhD
Economist
HNP Global Engagement
T  +1 (202) 473 0262
M  +1 (202) 597 8006
A 1776 G Street NW, G7-080, Washington, DC 20433 USA
[cid:image002.jpg@01D61E43.044B8AB0]

**Pull request merged**